relay-server/src/statsd.rs

```diff
@@ -973,3 +973,83 @@ impl<S: AsRef<str>> From<S> for PlatformTag {
         }
     }
 }
+
+pub enum SdkTag {
+    Ruby,
+    CocoaFlutter,
+    CocoaReactNative,
+    Cocoa,
+    Dotnet,
+    AndroidReactNative,
+    AndroidJava,
+    SpringBoot,
+    JavascriptBrowser,
+    Electron,
+    NestJs,
+    NextJs,
+    Node,
+    React,
+    Vue,
+    Native,
+    Laravel,
+    Symfony,
+    Php,
+    Python,
+    Other,
```

Comment:
If you change `Other` to `Other(String)` or `Other(&'a str)` we can still recover the original full name. We use something similar for metric units and envelope items already.